src/vault.js

```diff
@@ -170,6 +179,27 @@ var Page = function() {
     });
   }
 
+  self.apiTokenRevoke = function() {
+    $.ajax({
+      url: self.getUrl('auth/token/revoke-self'),
+      method: 'POST',
+      headers: self.getHeaders(),
+      success: self.apiTokenRevokeSuccess,
+      error: onError
+    });
+  }
+
+  self.getTokenInfo = function() {
+    if (self.token()) self.apiToken();
+    if (self.vaultTokenResponse()) {
+        var t = JSON.parse(self.vaultTokenResponse());
+        var id = t.id;
+        var name = t.display_name;
+        return name + '@' + id;
```

Comment:
I don't think we want to display the token in the main UI. You can click on the `System` dropdown and select `Token Information` to see the token and its details. How about just the name here instead?

![image](https://cloud.githubusercontent.com/assets/192336/17705819/ff44cb72-6396-11e6-8fe2-9ea9bc655957.png)